docs/app/pages/css/css-sections/icons/ux-icons/icon-preview/icon-preview.component.ts

```diff
@@ -0,0 +1,28 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+
+@Component({
+    selector: 'uxd-icon-preview',
+    templateUrl: './icon-preview.component.html',
+    styleUrls: ['./icon-preview.component.less'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class IconPreviewComponent {
+    @Input() name: string;
+    @Input() classname: string;
+    @Input() iconset: string;
+    @Input() iconSetClass: string;
+
+    constructor() {}
+
+    getUXComponentSnippet(): string {
+        return `<ux-icon name="${this.name}"></ux-icon>`;
+    }
+
+    getUXIconSnippet(): string {
+        return `<i class="ux-icon ${this.classname}"></i>`;
```

Comment:
Can't we use `iconSetClass` instead of hard coding `ux-icon` here, therefore eliminating the need for separate functions?

Also, I don't believe that the inputs ever change on this component (correct me if I am wrong) so you can generate the snippet strings once within `ngOnInit` and provide them to the view as properties.